0_portfolio_input_check_functions.R

```diff
@@ -6,8 +6,8 @@ read_raw_portfolio_file <- function(project_name) {
 
   input_path <- paste0(project_location, "/20_Raw_Inputs/")
 
-  csv_to_read <- list.files(path = input_path, pattern = paste0(project_name, "_Input.csv"))
-  txt_to_read <- list.files(path = input_path, pattern = paste0(project_name, "_Input.txt"))
+  csv_to_read <- list.files(path = input_path, pattern = "_Input.csv")
```

Comment:
A more robust pattern may be `"_Input[.]csv$"`

Reply:
👍🏻 those regexes can sneak up on you

Reply:
I'll go ahead and do the change. Here is my modes test

``` r
pattern <- "_Input[.]csv"
filename <- "whatever_Input.csv"
grepl(pattern, filename)
#> [1] TRUE
```

<sup>Created on 2021-02-17 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>